shacl12-overview/index.html

```diff
@@ -373,61 +373,211 @@ <h2>Introduction</h2>
-				<dd>defines SPARQL-related extensions of the SHACL</dd>
-				<dt><a href="https://w3c.github.io/data-shapes/shacl12-node-expr/">SHACL 1.2 Node Expressions</a></dt>
-				<dd>defines graph expressions used to determine focus nodes in SHACL</dd>
-				<dt><a href="https://w3c.github.io/data-shapes/shacl12-inf-rules/">SHACL 1.2 Inference Rules</a></dt>
-				<dd>defines SHACL's methods of rule-based inference</dd>
-				<dt><a href="https://w3c.github.io/data-shapes/shacl12-ui/">SHACL 1.2 UI</a></dt>
-				<dd>defines SHACL's use for User Interface generation</dd>
-				<dt><a href="https://w3c.github.io/data-shapes/shacl12-compact-syntax/">SHACL 1.2 Compact Syntax</a></dt>
-				<dd>defines an RDF syntax for expressing SHACL concepts</dd>
-				<dt><a href="https://w3c.github.io/data-shapes/shacl12-profiling/">SHACL 1.2 Profiling</a></dt>
-				<dd>defines the use of SHACL for profiling data, including SHACL data</dd>
-			</dl>
-
-			<h3>
-				SHACL 1.0
-			</h3>
-			<p><em>The original SHACL specifications, now deprecated in favour of SHACL 1.2 specifications, above.</em></p>
-			<dl>
-				<dt><a href="https://www.w3.org/TR/2017/REC-shacl-20170720/">SHACL</a></dt>
-				<dd>the original SHACL Shapes Constraint Language definition</dd>
-				<dt><a href="https://www.w3.org/TR/2017/NOTE-shacl-af-20170608/">SHACL Advanced Features</a></dt>
-				<dd>a Working Group Note about "features to define custom targets, annotation properties, user-defined functions, node expressions and rules"</dd>
-				<dt><a href="https://www.w3.org/TR/shacl-js/">SHACL Javascript Extensions</a></dt>
-				<dd>a Working Group Note about JavaScript-based extension mechanism for the Shapes Constraint Language</dd>
-			</dl>
-		</section>
-
-		<section id="x">
-			<h2>Section X</h2>
-			<p class="todo">TODO: This section will describe...</p>
+      <p>
+        The current version of SHACL is 1.2.
+      </p>
+      <section id="shacl-1.2">
+        <h3>
+          SHACL 1.2
+        </h3>
+        <dl>
+          <dt>SHACL 1.2 Overview (this document)</dt>
+          <dd>overviews the set of SHACL specifications</dd>
+          <dt><a href="https://w3c.github.io/data-shapes/shacl12-core/">SHACL 1.2 Core</a></dt>
+          <dd>defines the Core of SHACL</dd>
+          <dt><a href="https://w3c.github.io/data-shapes/shacl12-sparql/">SHACL 1.2 SPARQL Extensions</a></dt>
+          <dd>defines SPARQL-related extensions of the SHACL</dd>
+          <dt><a href="https://w3c.github.io/data-shapes/shacl12-node-expr/">SHACL 1.2 Node Expressions</a></dt>
+          <dd>defines graph expressions used to determine focus nodes in SHACL</dd>
+          <dt><a href="https://w3c.github.io/data-shapes/shacl12-inf-rules/">SHACL 1.2 Inference Rules</a></dt>
+          <dd>defines SHACL's methods of rule-based inference</dd>
+          <dt><a href="https://w3c.github.io/data-shapes/shacl12-ui/">SHACL 1.2 UI</a></dt>
```

Comment:
It seems odd to use the `UI` acronym here when all the other titles are fully spelled out.
```suggestion
          <dt><a href="https://w3c.github.io/data-shapes/shacl12-ui/">SHACL 1.2 User Interfaces</a></dt>
```